packages/fxa-graphql-api/src/gql/cart.resolver.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('#payments-cart - resolvers', () => {
 
 // import { Provider } from '@nestjs/common';
 // import { Test, TestingModule } from '@nestjs/testing';
-// import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+// import { MozLoggerService } from '@fxa/shared/mozlog';
```

Comment:
I don't understand why this is all commented out.

Reply:
It happened here: https://github.com/mozilla/fxa/pull/15685. I don't know why either. Line 13 just got caught up by a find and replace...